pkg/plugin/v2/plugin.go

```diff
@@ -25,7 +26,7 @@ const (
 	pluginVersion = "v2.0.0"
 )
 
-var supportedProjectVersions = []string{"2"}
+var supportedProjectVersions = []string{config.Version2, config.Version3}
```

Comment:
If the plugins will be supported for v2 why bump v3?

Reply:
Plugins are supported for all kubebuilder project versions. The Go plugin v2.0.0 supports project versions 2 and 3.

Reply:
You can close this one as solved.